`bin/setup` script fails then running make command, it can't find `imports/plugins/custom` under the reaction folder.

Issue tracker is **ONLY** used for reporting bugs. NO NEW FEATURE REQUESTS ACCEPTED. For new features, add an issue in our [Request Feature Request](https://github.com/reactioncommerce/reaction-feature-requests) repository.

Are you looking for help with getting started on Reaction? Please visit our [Reaction documentation](https://docs.reactioncommerce.com/reaction-docs/master/getting-started-developing-with-reaction).

## Prerequisites
* [x] Are you running the latest version?
* [x] Are you able to consistently reproduce the issue?
* [x] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)

## Issue Description
`bin/setup` script fails then running make command, it can't find `imports/plugins/custom` under the reaction folder.

## Steps to Reproduce
Preconditions:
This one might be hard to reproduce as it depends on the setup with `make init` for reaction-platform having failed once and having more subrepos than the stock install

1. With preconditions met, run `make` or `make init` again to try to finalize the project setup
2. Look output from the make command.

```
 ⇒  make
Error response from daemon: network with name auth.reaction.localhost already exists
Error response from daemon: network with name api.reaction.localhost already exists
Error response from daemon: network with name streams.reaction.localhost already exists
Error response from daemon: network with name routing.reaction.localhost already exists
Running pre-build hook script for reaction-hydra.
reaction-hydra post-project-start script invoked.
postgres uses an image, skipping
hydra-migrate uses an image, skipping
hydra uses an image, skipping
Running post-build hook script for reaction-hydra.
reaction-hydra post-build script invoked.
Starting reaction-hydra_postgres_1 ... done
Starting reaction-hydra_hydra-migrate_1 ... done
Starting reaction-hydra_hydra_1         ... done
Running post-project-start hook script for reaction-hydra.
reaction-hydra post-project-start script invoked.
Running pre-build hook script for sd-reaction.
sd-reaction post-project-start script invoked.
find: imports/plugins/custom: No such file or directory
/Users/user/reaction-platform/sd-reaction/bin/setup: line 27: port: No such file or directory
make: *** [prebuild-sd-reaction] Error 1   

```

## Possible Solution



### Versions
Node: 8.11.1
NPM: 6.0.0
Reaction CLI: 0.29.0
Docker: 18.09.2

--- a/bin/setup
+++ b/bin/setup
@@ -1,5 +1,18 @@
 #!/usr/bin/env bash
 
+# Please Use Google Shell Style: https://google.github.io/styleguide/shell.xml
+
+# ---- Start unofficial bash strict mode boilerplate
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit  # always exit on error
+set -o errtrace # trap errors in functions as well
+set -o pipefail # don't ignore exit codes when piping output
+set -o posix    # more strict failures in subshells
+# set -x          # enable debugging
+
+IFS="$(printf "\n\t")"
+# ---- End unofficial bash strict mode boilerplate
+
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 env_file=${__dir}/../.env
 
@@ -16,8 +29,8 @@ function add_new_env_vars() {
     chmod 0600 "${env_file}"
   }
 
-  find . imports/plugins/custom -name .env.example -type f -print0 |
-    xargs -0 grep -Ehv '^\s*#' |
+  find . -name .env.example -type f -print0 |
+    xargs -0 grep -Ehv '^\s*#' || true |
     {
       while IFS= read -r var; do
         if [[ -z "${var}" ]]; then

